doc/source/cluster/kubernetes/user-guides/observability.md

```diff
@@ -21,13 +21,56 @@ kubectl logs $KUBERAY_OPERATOR_POD -n $YOUR_NAMESPACE | tee operator-log
+| Suspended | The state will be set to `Suspended` when `Spec.Suspend` is set to true and all Pods in the cluster have been deleted.                             |
+
+
+
+#### RayCluster `.Status.Conditions`
+
+Although `Status.State` can represent the cluster situation, it is still only a single field. By enabling the feature gate `RayClusterStatusConditions` on the KubeRay v1.2.1, you can access to new `Status.Conditions` for more detailed cluster history and states.
+
+:::{warning}
+`RayClusterStatusConditions` is still an alpha feature and may undergo changes in the future.
+:::
+
+If you deployed KubeRay with Helm, then enable the `RayClusterStatusConditions` gate in the `featureGates` of your Helm values.
+
+```bash
+helm upgrade kuberay-operator kuberay/kuberay-operator --version 1.2.1 \
+  --set featureGates\[0\].name=RayClusterStatusConditions \
+  --set featureGates\[0\].enabled=true
+```
+
+Or, just make your kuberay operator executable run with `--feature-gates=RayClusterStatusConditions=true` argument.
+
+| Type                     | Status | Reason                         | Description                                                                                                          |
+|--------------------------|--------|--------------------------------|----------------------------------------------------------------------------------------------------------------------|
+| RayClusterProvisioned    | True   | AllPodRunningAndReadyFirstTime | Once all the Pods in the cluster are ready, this condition will be set to `True` and will remain `True` even if some Pods fail later. |
+|                          | False  | RayClusterPodsProvisioning     |                                                                                                                      |
+| RayClusterReplicaFailure | True   | FailedDeleteAllPods            | This condition will be set to `True` when there is a reconciliation error, otherwise the condition will be cleared.            |
+|                          | True   | FailedDeleteHeadPod            | Please refer to the `Reason` and the `Message` of the condition for more detailed debugging information.                 |
+|                          | True   | FailedCreateHeadPod            |                                                                                                                      |
+|                          | True   | FailedDeleteWorkerPod          |                                                                                                                      |
+|                          | True   | FailedCreateWorkerPod          |                                                                                                                      |
+| HeadPodReady             | True   | HeadPodRunningAndReady         | This condition will be `True` only if the HeadPod is currently ready; otherwise, it will be `False`.                         |
```

Comment:
```suggestion
| HeadPodReady             | True   | HeadPodRunningAndReady         | This condition is `True` only if the HeadPod is currently ready; otherwise, it is `False`.                         |
```